drivers/power/lt3074/lt3074.h

```diff
@@ -0,0 +1,275 @@
+#define LT3074_WRITE_PROTECT			(LT3074_R1B | 0x10)
+#define LT3074_CAPABILITY			(LT3074_R1B | 0x19)
+#define LT3074_VOUT_MODE			(LT3074_R1B | 0x20)
+#define LT3074_VOUT_OV_WARN_LIMIT		(LT3074_R2B | 0x42)
+#define LT3074_VOUT_UV_WARN_LIMIT		(LT3074_R2B | 0x43)
+#define LT3074_IOUT_OC_FAULT_LIMIT		(LT3074_R2B | 0x46)
+#define LT3074_IOUT_OC_FAULT_RESPONSE		(LT3074_R1B | 0x47)
+#define LT3074_OT_WARN_LIMIT			(LT3074_R2B | 0x51)
+#define LT3074_VIN_OV_WARN_LIMIT		(LT3074_R2B | 0x57)
+#define LT3074_VIN_UV_WARN_LIMIT		(LT3074_R2B | 0x58)
+#define LT3074_STATUS_BYTE			(LT3074_R1B | 0x78)
+#define LT3074_STATUS_WORD			(LT3074_R2B | 0x79)
+#define LT3074_STATUS_VOUT			(LT3074_R1B | 0x7A)
+#define LT3074_STATUS_IOUT			(LT3074_R1B | 0x7B)
+#define LT3074_STATUS_INPUT			(LT3074_R1B | 0x7C)
+#define LT3074_STATUS_TEMPERATURE		(LT3074_R1B | 0x7D)
+#define LT3074_STATUS_CML			(LT3074_R1B | 0x7E)
+#define LT3074_STATUS_MFR_SPECIFIC		(LT3074_R1B | 0x80)
+#define LT3074_READ_VIN				(LT3074_R2B | 0x88)
+#define LT3074_READ_VOUT			(LT3074_R2B | 0x8B)
+#define LT3074_READ_IOUT			(LT3074_R2B | 0x8C)
+#define LT3074_READ_TEMPERATURE_1		(LT3074_R2B | 0x8D)
+#define LT3074_REVISION				(LT3074_R1B | 0x98)
+#define LT3074_IC_DEVICE_ID			(LT3074_R6B | 0xAD)
+#define LT3074_IC_DEVICE_REV			(LT3074_R2B | 0xAE)
+
+/* Manufacturer registers */
+#define LT3074_MFR_MARGIN			(LT3074_R1B | 0xC4)
+#define LT3074_MFR_READ_VBIAS			(LT3074_R2B | 0xC6)
+#define LT3074_MFR_BIAS_OV_WARN_LIMIT		(LT3074_R2B | 0xC7)
+#define LT3074_MFR_BIAS_UV_WARN_LIMIT		(LT3074_R2B | 0xC8)
+#define LT3074_MFR_IOUT_MIN_WARN_LIMIT		(LT3074_R2B | 0xC9)
+#define LT3074_MFR_SPECIAL_ID			(LT3074_R2B | 0xE7)
+#define LT3074_MFR_DEFAULT_CONFIG		(LT3074_R1B | 0xF5)
+#define LT3074_MFR_RAIL_ADDRESS			(LT3074_R1B | 0xFA)
+#define LT3074_MFR_RESET			(LT3074_R0B | 0xFD)
+
+/* PMBus-specific parameters */
+#define LT3074_CRC_POLYNOMIAL			0x7
+#define LT3074_MARGIN_HIGH_MSK			0xF0
+#define LT3074_MARGIN_LOW_MSK			0x0F
+#define LT3074_OPERATION_ACCESS_MSK		(NO_OS_GENMASK(5,4) | NO_OS_BIT(7))
+#define LT3074_SPECIAL_ID_VALUE			0x1C1D
+
+/* Status types masks */
+#define LT3074_STATUS_BYTE_TYPE_MSK		0x01
+#define LT3074_STATUS_VOUT_TYPE_MSK		0x02
+#define LT3074_STATUS_IOUT_TYPE_MSK		0x04
+#define LT3074_STATUS_INPUT_TYPE_MSK		0x08
+#define LT3074_STATUS_TEMP_TYPE_MSK		0x10
+#define LT3074_STATUS_CML_TYPE_MSK		0x20
+#define LT3074_STATUS_MFR_SPECIFIC_TYPE_MSK	0x40
+#define LT3074_STATUS_WORD_TYPE_MSK		0x80
+#define LT3074_STATUS_ALL_TYPE_MSK		0xFF
+
+/* LINEAR data format params */
+#define LT3074_LIN11_MANTISSA_MAX		1023000L
+#define LT3074_LIN11_MANTISSA_MIN		511000L
+#define LT3074_LIN11_EXPONENT_MAX		15
+#define LT3074_LIN11_EXPONENT_MIN		-15
+#define LT3074_LIN11_MANTISSA_MSK		NO_OS_GENMASK(10,0)
+#define LT3074_LIN11_EXPONENT_MSK		NO_OS_GENMASK(15,11)
+#define LT3074_LIN11_EXPONENT(x)		((int16_t)(x) >> 11)
+#define LT3074_LIN11_MANTISSA(x)		(((int16_t)((x & 0x7FF) << 5)) >> 5)
+#define LT3074_LIN16_EXPONENT			-13
```

Comment:
These could be moved to the source file, since they're not meant to be used directly